src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/Frame.FeatureCollection.cs

```diff
@@ -202,6 +203,28 @@ string IHttpRequestIdentifierFeature.TraceIdentifier
             set => TraceIdentifier = value;
         }
 
+        long? IHttpMaxRequestBodySizeFeature.MaxRequestBodySize
+        {
+            get => MaxRequestBodySize;
+            set
+            {
+                if (HasStartedConsumingRequestBody)
```

Comment:
How does the application check this to make sure it's safe to do to avoid throwing?